libs/h3/src/test/java/org/elasticsearch/h3/ParentChildNavigationTests.java

```diff
@@ -132,13 +133,23 @@ private void assertHexRing(int res, String h3Address, String[] children) {
     public void testNoChildrenIntersecting() {
         String[] h3Addresses = H3.getStringRes0Cells();
         String h3Address = RandomPicks.randomFrom(random(), h3Addresses);
-        for (int i = 1; i <= H3.MAX_H3_RES; i++) {
+        // Once testIssue91915 is fixed, put upper limit of the loop to H3.MAX_H3_RES
+        for (int i = 1; i <= 10; i++) {
             h3Addresses = H3.h3ToChildren(h3Address);
             assertIntersectingChildren(h3Address, h3Addresses);
             h3Address = RandomPicks.randomFrom(random(), h3Addresses);
         }
     }
 
+    public void testIssue91915() {
+        GeoPolygon polygon1 = getGeoPolygon("8cc373cb54069ff");
+        GeoPolygon polygon2 = getGeoPolygon("8cc373cb54069ff");
```

Comment:
The h3 code is identical, so these two polygons are identical (same H3 cell). Perhaps you meant one of them to be different?

Reply:
That was a copy/paste error, I updated it to add the right cell. True that the original issue cannot be reproduced anymore, no idea why but here is another error if you want to have a look:

```
./gradlew ':libs:elasticsearch-h3:test' --tests "org.elasticsearch.h3.ParentChildNavigationTests.testNoChildrenIntersecting" -Dtests.seed=BD263AFF9F3817AA -Dtests.locale=ca-ES -Dtests.timezone=America/Merida -Druntime.java=1
```